docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,124 @@
+# Release 381 (11 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
```

Comment:
@arhimondr can you confirm the wording here .. and also confirm that this really does not need any docs??

Reply:
Extra space before period.